### PR TITLE
Added WINNF.FT.S.GRA.19

### PR DESCRIPTION
**Pull request by sushantparkhi**

### CODE REVIEW

File: src/harness/sas_interface.py [hunk 1]
```diff
@@ -371,4 +371,14 @@ def AssertContainsRequiredFields(self, schema_filename, response):
       response: A dictionary containing the response to validate for required
       fields using the schema.
     """
+    pass
+
+  @abc.abstractmethod
+  def CalculateDistance(self, lat1, lon1, lat2, lon2):
```
Comment (sushantparkhi) [inline src/harness/sas_interface.py]: How about adding it in the util.py?

File: src/harness/testcases/grant_testcase.py [hunk 2]
```diff
@@ -378,6 +378,53 @@ def test_WINNF_FT_S_GRA_17(self):
...
+    fss_zone_0 = json.load(
+      open(os.path.join('testcases', 'testdata', 'fss_zone_0.json')))
+    distance = self.CalculateDistance(device_a['installationParam']['latitude'],
+                                      device_a['installationParam']['longitude'],
+                                      fss_zone_0['deploymentParam'][0]['installationParam']['latitude'],
+                                      fss_zone_0['deploymentParam'][0]['installationParam']['longitude'])
+
+    # CBSD is within the 150km protection zone of FSS
+    self.assertLessEqual(distance, 150)
+    self._sas_admin.InjectFss({'record': fss_zone_0})
```
Comment (sushantparkhi) [inline src/harness/testcases/grant_testcase.py]: It's possible just to have FSS to create interference, but the CBSD has to be very close to FSS or by putting the FSS and CBSD within a very specific topology. But, I think it's probably best to have FSS with WISP to create Interference.

File: src/harness/sas_interface.py (hunk 1 above)
Comment (kate-harrison) [inline src/harness/sas_interface.py]: Sounds good to me :)

File: src/harness/testcases/grant_testcase.py (hunk 2 above)
Comment (kate-harrison) [inline src/harness/testcases/grant_testcase.py]: Sounds good. We might want to shoot a note to Kambiz that this test case is currently missing a step.

**Comment by sushantparkhi** [comment]
Co-located CBSD with FSS to generate 400 responseCode. This is also updated in the recent Test Spec.

**Comment by kate-harrison** [comment]
FYI GRA.19 is DEPRECATED.

**Pull request closed without merging**